### PR TITLE
Explain how team mapping work

### PR DESCRIPTION
**Pull request by juli1**
### What does this PR do? What is the motivation?

Explain to customer how team mapping work for static analysis.

### CODE REVIEW

File: content/en/code_analysis/faq/_index.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Datadog clones your code, runs the static analyzer to perform Static Analysis an
 The benefit of Datadog-hosted scans is that no configuration is needed in your CI pipeline(s) to use Code Analysis.
 
 
-### How are services and teams associated with code violations and libraries?
+### How are services are associated with code violations and libraries?
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:27]:
```suggestion
### How are services associated with code violations and libraries?
```

File: content/en/code_analysis/faq/_index.md [hunk 2]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
 in the path, the service name the closest to the filename is selected.
 
 
+### How are teams associated with code and libraries?
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:83]:
```suggestion
### How are teams associated with code violations and libraries?
```

File: content/en/code_analysis/faq/_index.md [hunk 3]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+
+Teams are associated either by getting the team associated with the service or by using
+the `CODEOWNERS` file and its mapping within Datadog.
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:86]:
```suggestion
Teams are associated by either getting the team associated with the service or using
the `CODEOWNERS` file and its mapping within Datadog.
```

File: content/en/code_analysis/faq/_index.md [hunk 4]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+
+When Datadog detects the service for your code violation or library, it automatically
+associates the team attached to the service. For example, if the file `domains/ecommerce/apps/myservice/foo.py`
+is associated with `myservice`, then the team of `myservice` will be associated to any violation
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:90]:
```suggestion
is associated with `myservice`, then the team `myservice` will be associated to any violation
```

File: content/en/code_analysis/faq/_index.md [hunk 5]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+for this file.
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:91]:
```suggestion
detected in this file.
```

File: content/en/code_analysis/faq/_index.md [hunk 6]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+
+If no service or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:93]:
```suggestion
If no services or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
```

File: content/en/code_analysis/faq/_index.md [hunk 7]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+file on your repository. The `CODEOWNERS` file associate file ownership to team on your Git provider. You need
```
Comment (amaanq) [inline content/en/code_analysis/faq/_index.md:94]:
```suggestion
file in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```

**Review by amaanq — APPROVED** [review]

File: content/en/code_analysis/faq/_index.md [hunk 8]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
...
+Teams are associated by either getting the team associated with the service or using
+the `CODEOWNERS` file and its mapping within Datadog.
+
+When Datadog detects the service for your code violation or library, it automatically
+associates the team attached to the service. For example, if the file `domains/ecommerce/apps/myservice/foo.py`
+is associated with `myservice`, then the team `myservice` will be associated to any violation
+detected in this file.
+
+If no services or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+file in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```
Comment (drichards-87) [inline content/en/code_analysis/faq/_index.md:94]:
```suggestion
If no services or teams are found, Datadog uses the `CODEOWNERS` [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```

Comment (drichards-87) [reply content/en/code_analysis/faq/_index.md:94]: Can we move the destination link to the bottom of the file? Inline links such as this one aren't compatible with our translation workflows.

File: content/en/code_analysis/faq/_index.md [hunk 9]
```diff
@@ -80,6 +80,20 @@ the name of the service is part of the path. If two services are present
 in the path, the service name the closest to the filename is selected.
 
 
+### How are teams associated with code violations and libraries?
+
+Teams are associated by either getting the team associated with the service or using
+the `CODEOWNERS` file and its mapping within Datadog.
```
Comment (drichards-87) [inline content/en/code_analysis/faq/_index.md:86]:
```suggestion
```

Comment (drichards-87) [reply content/en/code_analysis/faq/_index.md:86]: I think we could remove this sentence and still maintain the flow/readability.

**Review by drichards-87 — CHANGES_REQUESTED** [review]

File: content/en/code_analysis/faq/_index.md [hunk 10]
```diff
@@ -80,6 +80,17 @@ the name of the service is part of the path. If two services are present
...
 
+### How are teams associated with code violations and libraries?
+
+When Datadog detects the service for your code violation or library, it automatically
+associates the team attached to the service. For example, if the file `domains/ecommerce/apps/myservice/foo.py`
+is associated with `myservice`, then the team `myservice` will be associated to any violation
+detected in this file.
+
+If no services or teams are found, Datadog uses the `CODEOWNERS` [file][8]
+file in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```
Comment (drichards-87) [inline content/en/code_analysis/faq/_index.md:91]:
```suggestion 
in your repository. The `CODEOWNERS` file determines which team owns a file in your Git provider. You need
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**